internal/imagedefinition/README.rst

```diff
@@ -111,6 +111,13 @@ The following specification defines what is supported in the YAML:
              # An alternative branch to use while retrieving seeds
              # from a git or bzr source.
              branch: <string> (optional)
+          # Whether to write the sources list as Deb822 formatted entries in 
```

Comment:
Feels like an indentation mismatch?

Reply:
Yup! I really do not like YAML for this. *Especially* when the indentation is **one single space**. I will fix this.